src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -422,6 +422,7 @@ public interface IStartConfiguration
             IAuthProvider AuthProvider { get; set; }
             IEnumerable<IExtension> Extensions { get; set; }
             TaskProcessMode ProcessMode { get; set; }
+            bool HeadlessMode { get; set; }
```

Comment:
Thanks for that info, that's definitely not good.

I don't think putting it in the type and not the interface will help as DynamoModel Start and constructor take IStartConfiguration.

Looks like another possibility would be for Reach to specify an IUpdateManager object that does nothing.